src/Tribe/Attendee_Registration/Modal.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Attendee Registration Modal class
+ *
+ * @since TBD
+ */
+class Tribe__Tickets__Attendee_Registration__Modal {
+
+
+	public function hook() {
+
+		add_filter( 'tribe_events_tickets_attendee_registration_modal_content', [ $this, 'modal_cart_template' ], 10, 2 );
+	}
+
+	function modal_cart_template( $content, $template_obj ) {
```

Comment:
This relies on being in a class, so it works fine in the Gutenberg Ticket Block, but not in the classic editor.
I created this ticket to address the issue: #133180